lisa/nic.py

```diff
@@ -284,6 +310,15 @@ def reload(self) -> None:
         self.nics.clear()
         self._initialize()
 
+    def check_sriov_enabled(self, sriov_enabled: bool) -> None:
```

Comment:
Is this method needed? As we discussed, the sriov is a technology. It shouldn't be used in tests.

Reply:
I'll check the method name, it is needed when there is VF nic paired with synthetic nic, VF nic need time to be initialized and paired with synthetic nic

Reply:
We should avoid to use sriov, vf. It should use `pci`.